README.md

```diff
@@ -74,25 +74,21 @@ ember install ember-frost-demo-components
   This generates the following files
   ```
   tests/dummy
-    └── files
-      └── my-demo.js
-      └── my-demo.md
     └── app/pods
       └── my-demo
         └── controller.js
         └── route.js
         └── template.hbs
+        └── README.md
   ```
 
-5. Update `my-demo.js` and `my-demo.md` with the source code and documentation.
-
-6. Update `router.js` with the new routes.
+5. Update `router.js` with the new routes.
```

Comment:
I suppose.  I didn't want to muck around people's codez but I guess `ember-cli` does this already so I'll look at that.